core/src/components/textarea/test/states/textarea.e2e.ts

```diff
@@ -45,4 +45,30 @@ configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screensh
       await expect(container).toHaveScreenshot(screenshot(`textarea-focused`));
     });
   });
+
+  test.describe(title('textarea: states'), () => {
+    test('should render readonly textarea correctly', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-textarea fill="outline" label="Email" value="hi@ionic.io" readonly="true"></ion-textarea>
+      `,
+        config
+      );
+
+      const textarea = page.locator('ion-textarea');
+      await expect(textarea).toHaveScreenshot(screenshot(`textarea-readonly`));
+    });
+
+    test('should render disabled textarea correctly', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-textarea fill="outline" label="Email" value="hi@ionic.io" disabled="true"></ion-textarea>
+      `,
+        config
+      );
+
+      const textarea = page.locator('ion-textarea');
+      await expect(textarea).toHaveScreenshot(screenshot(`textarea-disabled`));
+    });
+  });
```

Comment:
Remove this and use the existing disabled and readonly.

```suggestion
```

On line 4: https://github.com/ionic-team/ionic-framework/blob/7be7c08cb037c94d112bd33874cdbc086ec9da43/core/src/components/textarea/test/states/textarea.e2e.ts#L4

change it to

```diff
- configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+ configs({ modes: ['ios', 'md', 'ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
```

By default, only `ios` and `md` are tested when no modes are passed to configs. By updating it this way, we can test all three.